src/features/dashboard/components/session-history-item/session-history-item.component.html

```diff
@@ -12,8 +13,7 @@ <h2>{{ item.projectId }}</h2>
 
   <ion-item>
     <div item-start>
-      <!-- Dev : the h2 should be a readable time ... ideally m/d/y-->
-      <p>{{ time }}</p>
+      <p>{{ formattedTime }}</p>
```

Comment:
For this i already made this change on my scoping-ui branch, i should create a PR only with this change. You don't need moment to format the date, we just need to use the date pipe from angular that handles this.

`{{ time | date: 'dd/MM/yyyy' }}`